tools/gen_parser/skulpt_parser_genarator.py

```diff
@@ -171,7 +171,7 @@ def visit_Opt(self, node: Opt) -> Tuple[str, str]:
         if call.endswith(","):
             return "opt", call
         else:
-            return "opt", f"{call} || 1"
+            return "opt", f"{call}, 1"
```

Comment:
This is the change that make it happen. In the python version they just have a , with no 1.